spec/lib/tic_tac_toe_spec.rb

```diff
@@ -4,7 +4,7 @@
   context "when the user starts the game" do
     it "should print a welcome message" do
       tic_tac_toe = TicTacToe.new;
-      expect { tic_tac_toe.printWelcome }.to output("Welcome to a game of Tic-Tac-Toe!\n").to_stdout
+      expect { tic_tac_toe.print_welcome }.to output("Welcome to a game of Tic-Tac-Toe!\n").to_stdout
```

Comment:
Unit tests should be testing the behavior and logic of the tested component. The logic behind `TicTacToe#print_welcome` is probably not outputting into console, but rather calling `CLI#put_string`